website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -202,6 +202,8 @@ git push -f upstream latest
 - Post to Kubernetes slack in #kubernetes-operators and #operator-sdk-dev.
 - In the [GitHub milestone][gh-milestones], bump any open issues to the following release.
 
+**Note**
+In case there are errors while building binaries for the released tag, you may have to revert the release. To do so, create a PR which reverts step [2][release_PR]. If there are changes in go modules of the underlying commit which caused the failure, it is suggested that we skip that particular version and release the next version.
```

Comment:
Is there an example of the go modules causing the failure? And why do the binaries builds fail?

Reply:
It isn't just binaries, it's any part of the build process.
```suggestion
In case there are non-transient errors while building the release job, you must
1. Revert the release PR. To do so, create a PR which reverts step [2][release_PR].
2. Fix what broke in the release branch.
3. Re-run the release with an incremented minor version to avoid Go module errors (ex. if v1.3.1 broke, then re-run the release as v1.3.2). Patch versions are cheap so this is not a big deal.
```